flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -56,7 +57,7 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
     public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         switch call.method as String {
         case "loadContexts":
-            loadContexts(result: result)
+            loadContexts(call, result: result)
```

Comment:
On Android we have a specific method call for loading image list, called `loadImageList`, can we reuse the same integration instead of returning it along with the `loadContexts`?

Reply:
We could, I actually started that and that realized you specifically mentioned to use "loadContexts" during our call so went that route. I'll make the update (this also applies to the other related comments)

Reply:
Ok, it was likely a communication issue, I explained that we already use `loadContexts` only on iOS, and `loadImageList` only on Android, but we could actually make the integration (loadImageList) for both and reuse it, sorry about that.

Reply:
done